pkg/controllers/disruption/validation.go

```diff
@@ -104,7 +107,7 @@ func (v *Validation) IsValid(ctx context.Context, cmd Command) (bool, error) {
 
 // ShouldDisrupt is a predicate used to filter candidates
 func (v *Validation) ShouldDisrupt(_ context.Context, c *Candidate) bool {
-	if c.Annotations()[v1alpha5.DoNotConsolidateNodeAnnotationKey] == "true" {
+	if c.Annotations()[v1beta1.DoNotDisruptAnnotationKey] == "true" {
```

Comment:
Shouldn't change this. Same reason as the other location. We need this backwards compat but should add a `TODO` for v1